imap/ical_support.c

```diff
@@ -516,7 +516,7 @@ static void multirrule_iterator_add(struct multirrule_iterator *iter,
     entry->icaliter = icaliter;
     entry->next = icalrecur_iterator_next(entry->icaliter);
     entry->recur = recur;
-    entry->recur->rscale = xstrdupnull(recur->rscale);
+    icalrecurrencetype_ref(recur);
```

Comment:
Hmm.  I misread this initially as icalrecurrencetype_unref().  We don't have a non-recur-by-ref version of this function.  We could make a macro which does nothing or #ifdef protect this.

Reply:
alright. I'll re-add the unref then.

Reply:
I mean, I'll re-add guarding the unref then.